apps/chat/src/store/application/application.epics.ts

```diff
@@ -0,0 +1,196 @@
+import { EMPTY, of } from 'rxjs';
+import { catchError, filter, map, switchMap } from 'rxjs/operators';
+
+import { combineEpics } from 'redux-observable';
+
+import { ApplicationService } from '@/src/utils/app/data/application-service';
+import { constructPath } from '@/src/utils/app/file';
+import { translate } from '@/src/utils/app/translation';
+import { ApiUtils } from '@/src/utils/server/api';
+
+import {
+  ApplicationListItemModel,
+  CreateApplicationModel,
+  DeleteApplicationAction,
+} from '@/src/types/applications';
+import { EntityType } from '@/src/types/common';
+import { AppEpic } from '@/src/types/store';
+
+import { UIActions } from '@/src/store/ui/ui.reducers';
+
+import { errorsMessages } from '../../constants/errors';
+
+import { ApplicationActions } from '../application/application.reducers';
+import { ModelsActions } from '../models/models.reducers';
+
+const createApplicationEpic: AppEpic = (action$) =>
+  action$.pipe(
+    filter(ApplicationActions.create.match),
+    switchMap(
+      ({
+        payload,
+      }: {
+        payload: {
+          applicationName: string;
+          applicationData: CreateApplicationModel;
+        };
```

Comment:
```suggestion
      }
```

You do not need to define a type here. The type will be defined automatically